### PR TITLE
Various fixes for the Rails template

### PR DESCRIPTION
**Pull request by Sam-Killgallon**
I recently set up ship and came across a few things that required debugging in order to get it running

1. Shipit uses the `redis-rails` gem for caching which doesn't support cache_versioning, so the app refuses to start. Rails ships with a `redis_cache_store` that does support this, but I didn't have the time to fully test it to know if its a drop in replacement, so I just disabled versioning for now https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-rediscachestore
Screenshot 2020-03-27 at 7 04 53 pm

2. I had some confusion getting Shipit to talk to Github because the provided secrets file used the same environment variable for the `app_id` and the `installation_id` even though they are different values

3. Pubsubstub seems to cause the whole app to hang after a little while (Discussed https://github.com/Shopify/shipit-engine/issues/1024). I appreciate in that thread the decision hadn't been made whether or not to make this the default, so can drop this commit if preferred.

4. A slightly confusing message was output during the templating, didn't cause any issues but thought it was worth clearing up. Rails seems to already supply the `ruby <version>` line in the Gemfile, and the message comes from `Thor` because it's already there https://github.com/erikhuda/thor/blob/da1e707627578eacbf7c6336ed1bb22fb3abfb86/lib/thor/actions/inject_into_file.rb#L63
Screenshot 2020-03-27 at 7 06 41 pm

Thank you for your hard work on this app and I'm looking forward to trying it out 👍

### CODE REVIEW

**Comment by casperisfine** [comment]
> 1. Shipit uses the `redis-rails` gem for caching which doesn't support cache_versioning, so the app refuses to start.

Yeah, we no longer need it. I simply forgot to remove it after the Rails 6.0 upgrade.

File: template.rb [hunk 1]
```diff
@@ -18,7 +18,9 @@
 worker: bundle exec sidekiq -C config/sidekiq.yml
 CODE
 
+environment 'Pubsubstub.use_persistent_connections = false'
 environment 'config.cache_store = :redis_store, Shipit.redis_url.to_s, { expires_in: 90.minutes }', env: :production
```
Comment (casperisfine) [inline template.rb]: Let's go all the way and use the `:redis_cache_store` shipped with Rails.

**Comment by Sam-Killgallon** [comment]
Thanks for the feedback. I'll apply the changes when I get a spare moment in the next couple of days

File: template.rb [hunk 2]
```diff
@@ -11,14 +11,16 @@
...
+gsub_file 'Gemfile', "# Use Redis adapter to run Action Cable in production", ''
+gsub_file 'Gemfile', "# gem 'redis'", "gem 'redis'"
 
 create_file 'Procfile', <<-CODE
 web: bundle exec rails s -p $PORT
 worker: bundle exec sidekiq -C config/sidekiq.yml
 CODE
 
-environment 'config.cache_store = :redis_store, Shipit.redis_url.to_s, { expires_in: 90.minutes }', env: :production
+environment 'Pubsubstub.use_persistent_connections = false'
```
Comment (casperisfine) [inline template.rb:22]: Good idea. That's probably a better default for most people.

**Comment by casperisfine** [comment]
Thanks!

**Pull request merged**